Fun Commands/Mini-Games/Dyno-RPG/divinecure.txt

```diff
@@ -0,0 +1,27 @@
+{delete}
+{silent}
+{require:{&Adventurer}}
+{require:serverMod}
+{require:{#divine-council}}
+{!role {&Adventurer} -Infected}
+{!role {&Adventurer} -Blighted}
+{!role {&Adventurer} -Festered}
+{!role {&Adventurer} -Bleeding}
+{!role {&Adventurer} -Cursed}
+{!role {&Adventurer} -Sacrificed}
+{!announce {#the-world-gate} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#the-crossroads} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#the-bloodfields} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#the-deathpit} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#castle-grounds} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#castle-courtyard} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#sacrificial-offerings} The Divine Council has spoken...
+A Divine blessing has been sent down on all Adventurers}
+{!announce {#divine-council} The Divine Council has spoken...
```

Comment:
Might want to add something for the divine council just to show who cured the player (only a suggestion)

Reply:
I left that out intentionally. It is simply meant to be a blessing sent down by the divine council. I figure users an update that one on their end if they desire.